modules/dns/main.tf

```diff
@@ -21,3 +21,20 @@ resource "ibm_cis_dns_record" "dns_records" {
   data      = ((each.value.data != null) && (each.value.type == "SRV")) ? merge(each.value.data, { "name" : lookup(each.value, "name", "") }) : lookup(each.value, "data", null)
   proxied   = lookup(each.value, "proxied", false)
 }
+
+##############################################################################
+# Import DNS Records
+##############################################################################
+resource "ibm_cis_dns_records_import" "dns_record_import" {
```

Comment:
resource name -- shouldn't be dns_records_import

Reply:
Made the changes